File: Atividade Avaliativa Final/Javascript/entity/torneio/TorneioVolei.js
import { Torneio } from './Torneio.js';

export class TorneioVolei extends Torneio {
  constructor(nome) {
    super(nome);
  }

  getModalidade() {
    return 'Volei';
  }

  getPosicoes() {
    return ['Levantador', 'Oposto', 'Central', 'Ponteiro', 'Líbero'];
  }
}

File: Atividade Avaliativa Final/Python/entity/torneio/torneio_volei.py


from entity.torneio.torneio import Torneio

class TorneioVolei(Torneio):
    def get_modalidade(self) -> str:
        return "Volei"

    def get_posicoes(self) -> list:
        return ["Levantador", "Oposto", "Ponteiro", "Central", "Líbero"]